src/openrct2/cmdline_sprite.c

```diff
@@ -284,7 +285,7 @@ static int get_palette_index(sint16 *colour)
 	return -1;
 }
 
-static bool sprite_file_import(const char *path, rct_g1_element *outElement, uint8 **outBuffer, int *outBufferLength, int mode)
+static bool sprite_file_import(const char *path, int x_offset, int y_offset, rct_g1_element *outElement, uint8 **outBuffer, int *outBufferLength, int mode)
```

Comment:
Use `sint16` instead of int. The `x_` and `y_offset` members of `rct_g1_element` are of type `sint16` too.